backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -126,7 +126,7 @@ private static String buildSpecificImageDiscriminator() {
     private static final int CASSANDRA_MEMORY = 1024;
 
     private static final String CASSANDRA_CONFIG_DIR = "$CASSANDRA_CONFIG";
-    private static final String JVM_OPTIONS = CASSANDRA_CONFIG_DIR + "/jvm-server.options";
+    private static final String JVM_OPTIONS = CASSANDRA_CONFIG_DIR + "/jvm.options";
```

Comment:
? Is it because https://github.com/apache/james-project/pull/1689 got merged? I would be careful to make sure it does not get reversed to this in the end... might wanna rebase to be sure :)

Reply:
Look at the full diff -> all good!